api/src/main/java/org/eclipse/microprofile/openapi/annotations/security/SecurityRequirementsSets.java

```diff
@@ -0,0 +1,53 @@
+/**
+ * Copyright (c) 2022 Contributors to the Eclipse Foundation
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.eclipse.microprofile.openapi.annotations.security;
+
+import static java.lang.annotation.ElementType.METHOD;
+import static java.lang.annotation.ElementType.TYPE;
+import static java.lang.annotation.RetentionPolicy.RUNTIME;
+
+import java.lang.annotation.Inherited;
+import java.lang.annotation.Retention;
+import java.lang.annotation.Target;
+
+/**
+ * Represents an array of security requirement sets that can be specified for the operation or at definition level. Only
+ * one of requirement sets needs be satisfied.
```

Comment:
> or at definition level

I know this phrase is in the equivalent `SecurityRequirements` annotation docs as well, but reading it makes it seem that this annotation is supported alongside `@OpenAPIDefinition` on an `Application` or `package-info`. What do you think?

Reply:
I think it's very imprecise about where it's supported.

Would it be better to list the places where it is supported? (which would be on a resource method, on a resource class and within `@OpenAPIDefinition#securitySet`)

Reply:
I think that makes sense.